src/include/planner/analyze_plan.h

```diff
@@ -58,9 +62,9 @@ class AnalyzePlan : public AbstractPlan {
   }
 
  private:
-  storage::DataTable* target_table_ = nullptr;
+  storage::DataTable *target_table_ = nullptr;
```

Comment:
While you're at it, can you remove this class body initialization? Again, it isn't wrong, but it's inconsistent.